common/include/fwd-window.h

```diff
@@ -36,18 +36,18 @@ int window_close(window *wind);
 int window_exists(window *wind);
 window *window_get_front();
 window *window_get_first();
-window *window_get_next(window &wind);
-window *window_get_prev(window &wind);
+static inline window *window_get_next(window &wind);
```

Comment:
`fwd-window.h` is intended only for forward declarations.  A `static inline` function must be defined in any translation unit where it is used, so the only reason to declare it would be to support calling it from before the definition is seen.  For this commit, I think it would be better to omit these declarations and rely on the definition in `window.h` to provide the declaration implicitly.